src/server/scripts/Spells/spell_holiday.cpp

```diff
@@ -42,75 +42,65 @@ enum SpellsPicnic
     SPELL_ROMANTIC_PICNIC_ACHIEV    = 45123, // Romantic Picnic periodic = 5000
 };
 
-class spell_love_is_in_the_air_romantic_picnic : public SpellScriptLoader
+class spell_love_is_in_the_air_romantic_picnic : public AuraScript
 {
-    public:
-        spell_love_is_in_the_air_romantic_picnic() : SpellScriptLoader("spell_love_is_in_the_air_romantic_picnic") { }
+    PrepareAuraScript(spell_love_is_in_the_air_romantic_picnic);
 
-        class spell_love_is_in_the_air_romantic_picnic_AuraScript : public AuraScript
-        {
-            PrepareAuraScript(spell_love_is_in_the_air_romantic_picnic_AuraScript);
+    void OnApply(AuraEffect const* /*aurEff*/, AuraEffectHandleModes /*mode*/)
+    {
+        Unit* target = GetTarget();
+        target->SetStandState(UNIT_STAND_STATE_SIT);
+        target->CastSpell(target, SPELL_MEAL_PERIODIC, false);
+    }
 
-            void OnApply(AuraEffect const* /*aurEff*/, AuraEffectHandleModes /*mode*/)
-            {
-                Unit* target = GetTarget();
-                target->SetStandState(UNIT_STAND_STATE_SIT);
-                target->CastSpell(target, SPELL_MEAL_PERIODIC, false);
-            }
+    void OnPeriodic(AuraEffect const* /*aurEff*/)
+    {
+        // Every 5 seconds
+        Unit* target = GetTarget();
 
-            void OnPeriodic(AuraEffect const* /*aurEff*/)
-            {
-                // Every 5 seconds
-                Unit* target = GetTarget();
-                Unit* caster = GetCaster();
+        // If our player is no longer sit, remove all auras
+        if (target->GetStandState() != UNIT_STAND_STATE_SIT)
+        {
+            target->RemoveAura(SPELL_ROMANTIC_PICNIC_ACHIEV);
+            target->RemoveAura(GetAura());
+            return;
+        }
 
-                // If our player is no longer sit, remove all auras
-                if (target->GetStandState() != UNIT_STAND_STATE_SIT)
-                {
-                    target->RemoveAura(SPELL_ROMANTIC_PICNIC_ACHIEV);
-                    target->RemoveAura(GetAura());
-                    return;
-                }
+        target->CastSpell(target, SPELL_BASKET_CHECK, false); // unknown use, it targets Romantic Basket
```

Comment:
triggered = false is default, so can be removed here and below

Reply:
thx, done ;)